app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java

```diff
@@ -679,6 +679,7 @@ private void postUICommand(Runnable aRunnable) {
     private void updateCandidates() {
         if (mInputConnection == null || !mCurrentKeyboard.supportsAutoCompletion()) {
             setAutoCompletionVisible(false);
+            updateSpecialKeyLabels();
             return;
```

Comment:
@MortimerGoro . I notice we also need to update key labels for English keyboard when doing switch from SymbolKeyboard although it does not support autocompletion.